module/VuFind/src/VuFind/Recommend/AbstractSearchObject.php

```diff
@@ -139,6 +165,21 @@ public function init($params, $request)
                 $lookfor,
                 $params->getOptions()->getHandlerForLabel($typeLabel)
             );
+
+            // Set any hidden filters configured for this search
+            if (!empty($this->iniSection)) {
+                $ini = $params->getOptions()->getSearchIni();
+                $config = $this->configManager->get($ini);
+                $iniSection = $this->iniSection;
+                try {
+                    $hiddenFilters = $config->$iniSection->toArray() ?? [];
```

Comment:
I believe that you could use `$config->{$this->iniSection}->toArray()` if you wanted to avoid the need to create a separate `$iniSection` variable. Feel free to leave as-is if you prefer it this way for readability, but thought I'd mention this approach in case you weren't already aware of it (it's something I rarely use but occasionally find handy).

Reply:
I was not aware of that syntax!  Done.